### PR TITLE
Home tab view

### PR DESCRIPTION
**Pull request by rajsam003**
We used **react-native-collapsible-tab-view** to add the tabs.

Todo: 
- Add margins on top
- Add tests to the Tabs

https://user-images.githubusercontent.com/36475005/166922226-df3c1697-b820-4e11-8180-76a4e5b730dd.mov

Co-author : @patrinoua

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -617,4 +617,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 699abadbf388cff0fe90b1125d246fbdc4070831
 
-COCOAPODS: 1.11.3
+COCOAPODS: 1.10.1
```
Comment (pvinis) [inline ios/Podfile.lock:620]: make sure you run `bundle install` in `ios`, to get the newer version,  @rajsam003

File: android/app/src/main/java/com/energy/MainApplication.java [hunk 2]
```diff
@@ -33,6 +35,11 @@ protected List<ReactPackage> getPackages() {
         protected String getJSMainModuleName() {
           return "index";
         }
+
+        @Override
+        protected JSIModulePackage getJSIModulePackage() {
+        return new ReanimatedJSIModulePackage();
```
Comment (brainbicycle) [inline android/app/src/main/java/com/energy/MainApplication.java:41]: could we indent this similar to above ☝️

File: package.json [hunk 3]
```diff
@@ -34,15 +34,19 @@
...
     "react-native-gesture-handler": "^1.10.3",
     "react-native-haptic-feedback": "^1.13.0",
     "react-native-linear-gradient": "^2.5.6",
+    "react-native-pager-view": "^5.4.15",
     "react-native-qrcode-svg": "^6.1.2",
+    "react-native-reanimated": "^2.8.0",
     "react-native-safe-area-context": "^3.3.2",
     "react-native-screens": "^3.9.0",
     "react-native-svg": "^12.1.1",
+    "react-native-tab-view": "^3.1.1",
```
Comment (pvinis) [inline package.json:49]: should we remove this then?

File: src/Scenes/HomeTabs/HomeTabs.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+export const HomeTabs = () => {
+  const selectedPartner = GlobalStore.useAppState((state) => state.activePartnerID)
+
+  if (!selectedPartner) {
+    return <SelectPartnerScreen />
+  }
+
+  return (
+    <Tabs.Container renderHeader={Header} headerHeight={HEADER_HEIGHT}>
+      <Tabs.Tab name="Artists">
```
[the remark this answers is not in the document]
Comment (pvinis) [reply src/Scenes/HomeTabs/HomeTabs.tsx:29]: yep, exactly. might be a good change :)

File: android/app/src/main/java/com/energy/MainApplication.java (hunk 2 above)
Comment (rajsam003) [reply android/app/src/main/java/com/energy/MainApplication.java:41]: Thanks for noticing. Sorry about that.

**Pull request merged**